pkg/subgraph/src/mappings/registry-community.ts

```diff
@@ -227,9 +228,25 @@ export function handleMemberActivatedStrategy(
   if (strategy.memberActive) {
     membersActive = strategy.memberActive!;
   }
+
   membersActive.push(memberAddress.toHexString());
   strategy.memberActive = membersActive;
   strategy.save();
+
+  const strategyConfigId = strategy.config;
+  const strategyConfig = CVStrategyConfig.load(strategyConfigId);
+
+  if (strategyConfig !== null) {
+    if (strategyConfig.pointSystem == BigInt.fromI32(0)) {
+      const memberStrategyId = `${memberAddress.toHexString()}-${strategyAddress.toHexString()}`;
+      let memberStrategy = new MemberStrategy(memberStrategyId);
+      memberStrategy.member = memberAddress.toHexString();
+      memberStrategy.strategy = strategyAddress.toHexString();
+      memberStrategy.totalStakedPoints = BigInt.fromI32(0);
+      memberStrategy.activatedPoints = event.params._pointsToIncrease;
+      memberStrategy.save();
```

Comment:
some special reason that only apply for pointSystem 0?

Reply:
Is the workaround that we talked the other day, for the others systems is done on the power Increase <https://github.com/1Hive/gardens-v2/blob/3a352f391ecd9356c4e72e44afa0de47d99bac1b/pkg/subgraph/src/mappings/cv-strategy.ts#L284-L298>